components/ArticleCard/index.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+
+const StyledArticleCard = styled.div`
+padding: 10px;
+  background-color: #858f87;
+  color: white;
+  border-radius: 10px;
+  align-items: center;
+  overflow: hidden;
+  background: linear-gradient(rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0.6)), url(${(
+    props
+  ) => props.image});
+  background-size: cover;
+
+}
+`;
+
+export default function ArticleCard({ article }) {
+  return (
+    <StyledArticleCard image={article.image}>
+      <h3>{article.title}</h3>
+      <p>{article.body.slice(0, 100)}...</p>
```

Comment:
you can also use css here instead. the css property is called `text-overflow: ellipsis` and the great thing about it is that it works for any line width and is not hard coded to 100 characters. Your approach is also valid if you want to display multiple lines before cutting of the string.